src/Messaging/RabbitMq/RabbitMqConnectionFactory.cs

```diff
@@ -23,7 +24,7 @@ public interface IRabbitMqConnectionFactory
         /// <param name="password">Password</param>
         /// <param name="virtualHost">Virtual host</param>
```

Comment:
Please update docs `<param>... ` to include new params. 

do we want to use bool for `useSSL` here instead of string?

Reply:
will do.
I have been thinking about using bool as the input of CreateChannel method directly as well. However this is would force us to have the logic to convert the useSSL setting from string to bool in both RabbitMqMessagePublisherService and RabbitMqSubscriberService instead of simply passing the value to RabbitMqConnectionFactory and converting in it. I will follow the approach you prefer for this, let me know.

Thanks!